app/front/scripts/directives/visualizations/barchart/index.js

```diff
@@ -37,6 +37,19 @@ ngModule.directive('barChartVisualization', [
           }
         }, true);
 
+        $scope.$on('babbage-ui.click',
+          function($event, component, item) {
+            var categoryName = component.chart.category(item['index']);
+            $event.stopPropagation();
+
+            if (categoryName.toLowerCase() !== 'others') {
```

Comment:
I understand it might not be possible (not easily, at least), but is there any easy way to avoid hardcoding the `others` string here?